lib/modules/manager/cpanfile/common.ts

```diff
@@ -0,0 +1,12 @@
+import { regEx } from '../../../util/regex';
+
+export function extractPerlVersion(txt: string): string | null {
+  const perlMatch = regEx(
+    /^\s*requires\s+['"]perl['"],\s*['"]?v?(?<currentValue>[^'"]+)['"]?;\s*$/gm
+  ).exec(txt);
+  return perlMatch?.groups?.currentValue ?? null;
+}
+
+export function formatContent(input: string): string {
+  return input.replace(regEx(/^ {2}/), '') + '\n'; //remove leading witespace and add a new line at the end
```

Comment:
```suggestion
  return `${input.trimStart()}\n`; //remove leading witespace and add a new line at the end
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart